samples/fleetprovisioning.py

```diff
@@ -343,12 +318,18 @@ def waitForRegisterThingResponse():
                 raise Exception('CreateKeysAndCertificate API did not succeed')
 
             registerThingRequest = iotidentity.RegisterThingRequest(
+<<<<<<< HEAD
```

Comment:
Looks good outside of merge related issues. Ping me for approval once it's fixed =D